build.properties.docker

```diff
@@ -0,0 +1,173 @@
+# DSpace build.properties
```

Comment:
As noted in today's meeting, let's move this to `[src]/src/main/docker/build.properties`